src/directives/domPlugin.ts

```diff
@@ -0,0 +1,104 @@
+export class MDOMPlugin {
+    public static attach<PluginType extends MElementPlugin<OptionsType>, OptionsType>(c: {
```

Comment:
Mettre un nom plus significatif que "c". Si c'est pour "component" à la limite "comp" ça fait. Il n'y a pas de règles formelles, mais souvent on se limite à une lettre dans les boucles for (`for (i in...`) ou pour un catch (`catch (e)...`)